package.json

```
@@ -2,23 +2,31 @@
   "name": "@portabletext/types",
   "version": "1.0.3",
   "description": "Shared TypeScript definitions for core Portable Text data structures",
-  "exports": "./dist/index.js",
-  "types": "./dist/index.d.ts",
   "type": "module",
+  "types": "./dist/index.d.ts",
+  "source": "./src/index.ts",
+  "module": "./dist/index.js",
+  "exports": {
+    ".": {
+      "types": "./dist/index.d.ts",
+      "source": "./src/index.ts",
+      "import": "./dist/index.js",
+      "default": "./dist/index.js"
+    }
+  },
   "sideEffects": false,
   "engines": {
     "node": "^12.20.0 || ^14.13.1 || >=16.0.0"
   },
   "files": [
     "dist",
-    "src",
-    "README.md"
+    "src"
   ],
   "scripts": {
     "docs": "typedoc",
     "test": "eslint . && npm run build && node test/test.js",
     "prepublishOnly": "npm run build && npm run test",
-    "build": "tsc --declaration"
+    "build": "pkg build"
```

I kinda prefer the `pkg-utils` binary as it is easier to understand where it is coming from and can't be confused with vercel's [pkg](https://github.com/vercel/pkg) or freebsd's [pkg](https://www.freebsd.org/cgi/man.cgi?query=pkg&sektion=&n=1)
```suggestion
    "build": "pkg-utils build"
```

---

Ok sure.